### PR TITLE
Fix codecov token

### PR DESCRIPTION
**Pull request by skjnldsv**

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

**Comment by jotoeri** [comment]
How about this one, @skjnldsv? This should make codecov reacting a bit more reliably, no?
Seems to me from the log, as it would only be to increase the fetch-depth to make it run? 🤔

File: phpunit.xml [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
-    </testsuites>
+<?xml version="1.0" encoding="utf-8" ?>
+<phpunit bootstrap="tests/bootstrap.php"
+         verbose="true"
+         colors="true"
+         timeoutForSmallTests="900"
+         timeoutForMediumTests="900"
+         timeoutForLargeTests="900">
+	<testsuite name='Forms app tests'>
+		<directory>./tests/unit</directory>
```
Comment (jotoeri) [inline phpunit.xml:9]: Uppercase `Unit` here for our case...?

**Comment by skjnldsv** [comment]
> How about this one, @skjnldsv? This should make codecov reacting a bit more reliably, no?

I didn't found the time to finish this, feel free to take over @jotoeri :)

**Comment by skjnldsv** [comment]
Rebased

**Comment by skjnldsv** [comment]
@jotoeri
`fixup! DO NOT MERGE Just a test` You can review and block a pr ;)

**Review by jotoeri — CHANGES_REQUESTED** [review]
Well then... 😉

File: composer.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
 		"cs:fix": "php-cs-fixer fix",
 		"cs:check": "php-cs-fixer fix --dry-run --diff",
 		"lint": "find . -name \\*.php -not -path './vendor/*' -print0 | xargs -0 -n1 php -l",
-		"phpunit": "phpunit"
+		"test:unit": "phpunit -c tests/phpunit.unit.xml",
+		"test:integration": "phpunit -c tests/phpunit.integration.xml"
```
Comment (skjnldsv) [inline composer.json:15]: Much cleaner! :heart_eyes:

**Comment by jotoeri** [comment]
Huhh - that was a hard one. 😅 
But seems to work now.
- I further moved the phpunit-stuff into the tests-dir. Probably one could move the codecov.yml too somewhen, but the codecov-action seems not to get that resolved currently.
- Codecov-comments should now only happpen if relevant and be a bit shorter

I'd probably wait with the merge until https://codecov.io/gh/nextcloud/forms/commit/047bee0a81a4e5f806e583481897143f6a4f0281 is done, but i assume that should be good.

**Review by jotoeri — APPROVED** [review]
🙈

**Pull request merged**